CMakeLists.txt

```diff
@@ -12,6 +12,11 @@ set(STUDIO_VERSION_PATCH 0)
 
 option(ENABLE_TCMALLOC "Enable use of tcmalloc library" OFF)
 
+if(VCPKG_TOOLCHAIN)
+	set(CMAKE_INSTALL_RPATH "\$ORIGIN/../lib")
+	add_definitions(-DSYNFIG_PORTABLE)
```

Comment:
Maybe move `include(SynfigVcpkg)` here too?

Reply:
I was hesitant to do that at first because it will require me to monkey patch `add_library` and `install` cmake functions. Adding it at the end of the file has the advantage that all the targets have been already defined.

In anyways, I now understand how this is done, so I'm going to do this.

Reply:
done